static/elements/chromedash-legend.html

```diff
@@ -42,14 +42,11 @@
     <section>
       <label>Example search queries</label>
       <ul class="queries">
-        <li><span>"&lt;30"</span>features that landed before 30</li>
-        <li><span>"&lt;=30"</span>features in 30</li>
-        <li><span>"=30"</span>features that landed in 30</li>
-        <li><span>"&gt;28"</span>features since 28</li>
-        <li><span>"behind a flag"</span>all experimental features</li>
-        <li><span>"origin trial"</span>all features available as origin trials</li>
-        <!-- <li><span>"category: CSS"</span>features in the category CSS</li> -->
-        <li><span>"owner: user@example.org"</span>features owned by user@example.org</li>
+        <li><span>"shipped_milestone&lt;30"</span>features that landed before 30</li>
```

Comment:
Notice change for versioning expressions. They use _<property_name>_ _<operator>_ _<integer>_